EIPS/eip-894.md

```diff
@@ -0,0 +1,36 @@
+## Preamble
+
+    EIP: 894
+    Title: Immutability Enforcement Proposal (IMP)
+    Author: Marius Kjærstad
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-21
+
+
+## Simple Summary
+Provide a standardized response against proposals for bailouts on the offical Ethereum repositories.
+
+
+## Abstract
+Proposals that advocate for any state changes to the Ethereum blockchain should not be merged into the official Ethereum repositories.
+
+
+## Motivation
+The issue of bailouts on the Ethereum blockchain is always controversial. This EIP attempts to raise the barriers for bailouts.
```

Comment:
The abstract refers _state changes_, no you use the term _bailouts_, please consider adding a short definition paragraph to the proposal to clarify what you are referring to.